.github/workflows/ci.yml

```diff
@@ -22,5 +22,23 @@ jobs:
         npm install
         npm run build:v2-compat
```

Comment:
Since we're doing multiple builds for the platforms, we'll need to build a separate `v2-compat` for each, I've updated the build scripts to enable this, `npm run build:v2-compat -- firefox/chrome/etc` should build the correct file for each platform.

Reply:
Rebase the pull request to test if it's working or not, the resulting artifacts should contain different `tabby2-background.js` files

MD5's for the files on a build on my local machine:
`tabby2-background.js` built with `npm run build:v2-compat -- firefox/webext` is `f9b0a6546f1fff2c3e0a4b0fb092e3c4`
`tabby2-background.js` built with `npm run build:v2-compat -- chrome` is `3d24fdb953014fb594208234763648d4`